src/assets/css/librarybrowser.css

```diff
@@ -1122,3 +1122,22 @@ div:not(.sectionTitleContainer-cards) > .sectionTitle-cards {
 .itemsViewSettingsContainer > .button-flat {
     margin: 0;
 }
+
+.detail-clamp-text {
+    overflow: hidden;
+    display: -webkit-box;
```

Comment:
why prefixed `display`? we have `gulp` building now that should add all needed prefixes

Reply:
It hasn't happened yet, that's all.

Reply:
What I meant is that we haven't quite updated our build process to remove all these again since the changes.